QUICKSTART.md

```diff
@@ -0,0 +1,279 @@
+# **KubeStellar** Quickstart
+
+## Required Packages:
+   - [docker](https://docs.docker.com/get-docker/)
+   - [kind](https://kind.sigs.k8s.io/)
+   - [kubectl](https://kubernetes.io/docs/tasks/tools/) (version range expected: 1.23-1.25)
+
+## Setup Instructions
+
+Table of contents:
+
+- [1. Install and run **KubeStellar**](#1-install-kubestellar-pre-requisites)
+- [2. Example deployment of Apache HTTP Server workload into two kind local clusters](#4-Example-deployment-of-nginx-workload-into-a-kind-local-cluster)
+  - [a. Stand up two kind clusters: florin and guilder](#a-Stand-up-two-kind-clusters-florin-and-guilder)
+  - [b. Create a KubeStellar Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)](#b-onboarding-the-clusters)
+  - [c. Onboarding the clusters](#b-onboarding-the-clusters)
+  - [d. Create and deploy the Apache Server workload into florin and guilder clusters](#c-Create-and-deploy-the-nginx-workload-into-florin-and-guilder-clusters)
+- [3. Cleanup the environment](#3-Cleanup-the-environment)
+
+
+This guide is intended to show how to quickly bring up a **KubeStellar** environment with its dependencies from a binary release.
+
+## 1. Install and run **KubeStellar**
+
+KubeStellar works in the context of kcp, so to use KubeStellar you also need kcp. Download the kcp and **KubeStellar** binaries and scripts into a `kubestellar` subfolder in your current working directory using the following command:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/francostellari/edge-mc/main/hack/kubestellar-bootstrap.sh) --kcp-version v0.11.0 --kubestellar-version v0.1.0 --folder . --create-folder
```

Comment:
(1) If the directory `.` exists then then `--create-folder` option does not help.  If the directory `.` does NOT exist then the `--create-folder` option does not help.  In short, with `--folder .` there is no benefit to `--create-folder`.

Reply:
Fixed